src/main/java/me/botsko/prism/listeners/PrismPlayerEvents.java

```diff
@@ -324,9 +324,15 @@ public void onPlayerInteract(final PlayerInteractEvent event) {
             // " Item in hand:" + player.getItemInHand().getTypeId() + ":" +
             // player.getItemInHand().getDurability());
 
+            int itemInHandId = player.getItemInHand().getTypeId();
```

Comment:
I guess it isn't exactly needed.

I've always seen things like multiple calls to find a value, when the value is unlikely to change between the two calls duplicate calls, and a waste of CPU time.  But that's just me, I guess other people don't care about a few wasted CPU cycles because of the large amount of processing power available in computers these days.  I like to grab a value once and store it locally, rather than grabbing it every time it's needed.

I've removed it to make you happy.